modules/core/client/components/Map/index.js

```diff
@@ -1,7 +1,8 @@
 // External dependencies
+import { debounce } from 'lodash';
```

Comment:
The _[whatever does the build - webpack?]_ probably optimizes the bundle size, however `import debounce from 'lodash/debounce'` would be the safe option for client side.

Reply:
I agree: `lodash/debounce` is best bet.

Otherwise need [https://github.com/lodash/babel-plugin-lodash](babel-plugin-lodash) to rewrite it for us...

Reply:
Yup! I'm just used to that being available from other codebases I work with. 😁 Should we just add the plugin tho? No big deal if you prefer not.